tianshou/data/batch.py

```diff
@@ -177,12 +180,21 @@ def __init__(self,
             elif _is_batch_set(batch_dict):
                 self.stack_(batch_dict)
         if len(kwargs) > 0:
-            self.__init__(kwargs, copy=copy)
+            self.__init__(kwargs, copy=copy)  # type: ignore
 
     def __setattr__(self, key: str, value: Any) -> None:
         """self.key = value"""
         self.__dict__[key] = _parse_value(value)
 
+    def __getattr__(self, key: str) -> Any:
+        """Return self.key. It is the out-of-the-box usage, but the "Any"
+        return type is needed for mypy type checking."""
+        return getattr(self.__dict__, key)
+
+    def __contains__(self, key: str) -> bool:
```

Comment:
Same

Reply:
This `__contains__` is the previous code, I only change the order.